### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by dleehr**
Creates a bare-bones Ember application for listing deliveries.

- Depends on https://github.com/Duke-GCB/D4S2/pull/99
- Copies DRF Adapter/authenticator/authorizer from bespin-ui. These will ultimately be extracted to a common library

In this version, only expectation is that users can login and see a listing of deliveries they are sending (or have sent). Delivery details and actions are not yet implemented.

### CODE REVIEW

**Comment by johnbradley** [comment]
I may have missed something but I am unable to login when using  `initial-version` from this repo and `ember-support` from D4S2. I see this in the D4S2 output:
```
Not Found: /api-auth-token/
WARNING:django.request:Not Found: /api-auth-token/
[02/Nov/2017 13:20:03] "OPTIONS /api-auth-token/ HTTP/1.1" 404 3010
```

D4S2 seems to think this should be `api-token-auth`:
https://github.com/Duke-GCB/D4S2/blob/1f5dd3577b619f0ee9e2721f9415e858c7776f3d/d4s2/urls.py#L15

**Comment by dleehr** [comment]
Can you double-check that your local copy of bespin-api is on `ember-support`? The [first commit](https://github.com/Duke-GCB/D4S2/pull/99/commits/51af318502a2d986693c4d27ab7fdfc358416840) in that branch changes api-token-auth to api-auth-token. The change is reflected in the link you posted 

https://github.com/Duke-GCB/D4S2/blob/1f5dd3577b619f0ee9e2721f9415e858c7776f3d/d4s2/urls.py#L15

**Comment by johnbradley** [comment]
Looks like I somehow had a branch with this same name locally. I think I have all the correct code now.

**Comment by johnbradley** [comment]
I started getting this error: 
```
Failed to load http://127.0.0.1:8000/api/v2/deliveries/: No 'Access-Control-Allow-Origin' 
header is present on the requested resource. Origin 'http://localhost:4200' is therefore 
not allowed access.
```
I had to manually add this: 
```
CORS_ORIGIN_WHITELIST = (
    'localhost:4200',
    '127.0.0.1:4200',
)
``` 
to `d4s2/settings.py`.

**Comment by dleehr** [comment]
Ah, yes I have that in my local settings.py, but since that's excluded from Git, it never made it to the branch. I've moved it to `settings_base.py`

File: app/authenticators/drf-token-authenticator.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        url: ENV.APP.API_URL + '/api-auth-token/',
+        type: 'POST',
+        data: JSON.stringify({
+          username: username,
+          password: password
+        }),
+        contentType: 'application/json',
+        dataType: 'json'
+      }).then((response) => {
+        Ember.run(function() { //what is this?
```
Comment (johnbradley) [inline app/authenticators/drf-token-authenticator.js]: Useless comment

File: app/authenticators/drf-token-authenticator.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          resolve({
+            token: response.token
+          });
+        });
+      }, (xhr, status) => {
+        let response = xhr.responseText || status;
+        Ember.run(function() { // again what?
```
Comment (johnbradley) [inline app/authenticators/drf-token-authenticator.js]: Useless comment

File: app/authorizers/drf-token-authorizer.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import Base from 'ember-simple-auth/authorizers/base';
+
+export default Base.extend({
+  session: Ember.inject.service('session'),
+  authorize(data, block) { // what is block?
```
Comment (johnbradley) [inline app/authorizers/drf-token-authorizer.js]: Useless comment

**Pull request merged**